src/WormholeJoin.sol

```diff
@@ -176,37 +178,44 @@ contract WormholeJoin {
             vat.slip(ilk, address(this), int256(amtToGenerate));
             vat.frob(ilk, address(this), address(this), address(this), int256(amtToGenerate), int256(amtToGenerate));
         }
-        daiJoin.exit(wormholeGUID.receiver, amtToTake - fee);
+        uint256 postFeeAmount = amtToTake - fee;
+        uint256 operatorFee = postFeeAmount * operatorFeePercentage / WAD;
```

Comment:
I'm wondering whether the operatorFee should be specified as a fixed fee instead. The operator must at least cover their gas cost but in some unfortunate scenario, the `amtToTake` may be less than anticipated (e.g. the tx was frontrun by a big withdrawal using up most of the available debt) and `operatorFee` could end up being not large enough to cover gas fees.

Reply:
Yeah so whether a fixed fee or a percentage we should expect the operator network to run all the calculations in advance to see what number makes sense and in the event of front-running the relay network tx should be failing.

I think I'm okay to switch this to a fixed fee as this calculus needs to be done on-chain in advance regardless.